Docs/compression.md

```diff
@@ -1,15 +1,16 @@
 
-**Imgbot** uses compression algorithms to optimize images, and by default `LosslessCompress()` is used.
-This means that while the file size is going down, the quality and dimensions remain intact.
+Imgbot uses compression algorithms to optimize images. While the file size is going down, the quality and dimensions remain intact.
 
-When configuring your setup, you can choose between non-aggressive `LosslessCompress()` and aggressive `LossyCompression()` compression.
+To provide the strong optimization, Imgbot uses multiple compression algorithms. 
 
-To provide the strong optimization, **Imgbot** utilizes multiple compression algorithms, and these are currently implemented:
+These algorithms are currently implemented:
  - [ImageMagick](http://www.imagemagick.org)
  - [Svgo](https://github.com/svg/svgo)
  - [MozJpeg](https://github.com/mozilla/mozjpeg)
 
 
-The images are compressed in place after cloning the repo, and once installed into any repo, **Imgbot** will run the `LosslessCompress()` routine on an ongoing basis and open PRs to keep your images optimized.
+By default, Imgbot uses the `LosslessCompress()` method. When setting up Imgbot, you can also switch to the aggressive `LossyCompression()` compression. <!-- what's the difference? -->
```

Comment:
question as comment. is the difference described here? https://imgbot.net/docs/#aggressive-compression

Reply:
```suggestion
By default, Imgbot uses the `LosslessCompress()` method. When setting up Imgbot, you can also switch to the aggressive `LossyCompression()` compression. Learn more about [the aggressive compression](#aggressive-compression).
```